### ISSUE TITLE
[Suggestion] Skybox brightness setting. 

### ISSUE THREAD

**Issue by CoasterBlock** [tsunamayo/Starship-EVO#6023]
Space is dark. Like really dark. Unless you hide from all lightsources, you wouldnt see the stars with the exeception of the closes and or brightest stars. 
I suggest adding a setting that would allow a "gamma" adjust for the skybox. Effectively allowing the players to adjust how bright the skybox would appear. On the lowest setting you would only see the brightest and closest stars. On the default setting, the skybox would appear as it does now. 

![](https://www.lunarskydiving.com/en/blog/how-far-away-is-the-moon/images/actual-earth-moon-distance.jpg)
Heres a photograph taken from a deep-space probe on its way to the outer solar system.

**Comment by ProPeach** [comment]
One thing that could help with this feeling is reducing the global illumination level (I think that's what it's called?) It used to be a bit lower than what it is now which gave some really rich, dark shadows, but it's brighter these days. I would really like to see that darkness come back, especially in space as it helps capture the vibe and give a better contrast to scenes. Of course in atmospheres there would be a bit more ligth scatter so the brighter shadows are less of an issue, but even there they could help with contrast.

**Comment by CoasterBlock** [comment]
Agreed, much more contrast is needed. Atmospheres feel odly bright even without ReShade. In addition, colored sunlight would also be a nice-to-have

**Comment by tsunamayo** [comment]
Yes I wanted to do another pass on that, add the sun color when close to the sun, dim light when far away, add more ambient when close to a planet and remove (so like you guys are asking) when from from any planet (or in the shadow of a planet).

@CoasterBlock Can you show me some screenshot of Reshade? You guys are using this?

**Comment by CoasterBlock** [comment]
@tsunamayo Sure, ReShade allows some postprocessing to the game. Below are some screenshots with the settings I use. 

_**Presets:**_
iMMERSE Launchpad (sorts data to make sure everything loads properly)
FilmicPass (oes some light color grading
NeoBloom (adds suble and ajustable bloom)
MXAO (ambient occlusion)
prod80_02_Cinetools_LUT (adds adjustable LUT, I use FilmicBlue_Contrast)
UnrealLens (adds a subtle and adjustable lens-flare)
SSR (screenspace reflections)
Letterbox (adds cinematic bars to the top and bottom of the screen, just for screenshots) 

Each of those shaders has many adjustments. The LUT mod, adds many different LUTs, the UnrealLens adds the unreal-engine lens flares which can be subtlely adjusted to look nice. Basically everyone of those shaders comes with dozens of options to toggle and finely adjust to what you like. 


**Without ReShade**
![20241007203858_1](https://github.com/user-attachments/assets/08a27176-c1d3-430c-822f-96180b513a02)
![20241007203943_1](https://github.com/user-attachments/assets/34c43c5f-178c-4255-bb63-f1f999c17823)
![20241007204014_1](https://github.com/user-attachments/assets/1fdb7d15-272d-42e3-b673-fce5740063d7)
![20241007203928_1](https://github.com/user-attachments/assets/330d16c6-de95-4f4e-9c42-3916cb97b9b0)


**With ReShade:** 
![20241007203947_1](https://github.com/user-attachments/assets/63afe7f4-c486-4870-ba7b-f8e058f8cbdb)
![20241007204003_1](https://github.com/user-attachments/assets/292f0f98-fbd6-4e9b-8665-db13f64af1d3)
![20241007204018_1](https://github.com/user-attachments/assets/998444e0-67f2-41c0-8d3d-d597503b7fc4)
![20241007203903_1](https://github.com/user-attachments/assets/2dc915f5-ac45-469d-a3a0-74f4868afa53)
![20241007203931_1](https://github.com/user-attachments/assets/2c273861-25d5-479c-b118-5629a471779e)

**Comment by ProPeach** [comment]
I don't have as much knowledge on graphics effects to utilize much of Reshade, but I do use it for a little bit of extra ambient occlusion as well as an anti ailiasing pass. You might have to open the images in new tabs and switch between them to see the difference, it isn't quite as stark as CoasterBlocks :D

Without Reshade
![711980_20241007202301_1](https://github.com/user-attachments/assets/8a72350f-1344-479c-9cc8-576d64fe2a48)


With some Reshade AA and AO
![711980_20241007202250_1](https://github.com/user-attachments/assets/d3a5c3f4-2b44-4bc1-9f5f-ab324d5fbebb)
